.yaydoc.yml

```diff
@@ -0,0 +1,35 @@
+metadata:
+  author: FOSSASIA
+  projectname: "Susi Server"
+  version: development
+build:
+  theme:
+    name: sphinx_fossasia_theme   
+  source: docs
+  logo: images/susi-logo-original.svg
+  subproject:
+    - url: https://github.com/fossasia/susi_android
+      source: docs
```

Comment:
it could've been omitted, but I don't think there is a need to do so. There is nothing wrong in keeping it verbose. It only makes the configuration more clear